.circleci/config.yml

```diff
@@ -48,10 +33,6 @@ jobs:
           key: node-v1-{{ checksum "composer.json" }}
           paths:
             - node_modules
```

Comment:
What's this line for?

Reply:
Saves the node cache for yarn installed dependencies